Sources/Views/ALKFormCell.swift

```diff
@@ -276,7 +281,11 @@ extension ALKFormCell: UITableViewDataSource, UITableViewDelegate {
             } else {
                 cell.accessoryType = .none
             }
-
+            if let check = viewModel?.isFormSubmitted() , check {
```

Comment:
rename the variable `check`

Reply:
Changed the name to checkForFormSubmitted